src/nodejs/lambda-handlers/workflow.js

```diff
@@ -11,6 +11,7 @@ async function createStep(step, stepName, rollbackInfo) {
   return db.workflow.createStep({
     step_name: stepName,
     type: step.type,
+    step_status_label: step.step_status_label,
```

Comment:
I think this was added after Davis' comment:
```
Fields in the step table need to be able to be edited using the workflow editor. Please reach out if you need help adding support for this.
```
however, this doesn't address the comment. For example, https://github.com/eosdis-nasa/earthdata-pub-api/blob/2d858252591f616cffe749993a6f040dfa83950b/src/nodejs/lambda-layers/database-util/src/query/workflow.js#L41 was not updated, so effectively this addition is never actually used.

Reply:
ohh I see, I will update the createStep function and push the changes soon.